src/user-research/choosing-and-using-analysis-and-synthesis-methods.md

```diff
@@ -40,6 +48,18 @@ Whatever approach we take, our analysis and synthesis goes through three broad s
 
    Working with our colleagues to agree what to do in response to the findings, including adapting our research activities, creating new design ideas, or recording issues to resolve.
 
+### Rich data and big data analysis
+
+Each research activity we do, typically produces one kind of data - notes or transcripts from interviews, videos from usability tests, completed boards or worksheets from workshops, responses from questionnaires.
+
+On longer projects, through many different research activities, with different groups of participants, we will produce a rich collection of different kinds of data and initial findings.
+
+On these projects an important part of our work is combining these to create a detailed, coherent and compelling picture of the most important things we have learned.
```

Comment:
@johnwaterworth I think this true, however I think there is something missing here. In longer projects we can do triangulation, which is not only about creating a detail, coherent and compelling picture, but also and most importantly about increasing credibility and identifying contradictions, gaps and biases. Do you think you could add that here and then refer to the triangulation method?

Reply:
@johnwaterworth  And, could we add this link to the list of recommended guidance for this section? https://www.nngroup.com/articles/triangulation-better-research-results-using-multiple-ux-methods/

Reply:
Thanks for those thoughts. Yes, this is a good place to mention triangulation. Can you have a look at the revised text in the new pull request.